lib/rules/declaration-block-no-redundant-longhand-properties/index.js

```diff
@@ -93,6 +93,7 @@ const rule = (primary, secondaryOptions) => {
 						ruleName,
 						result,
 						node: decl,
+						word: decl.prop,
```

Comment:
[note] If not specify `word`, `endColumn` would point to the end of the declaration's value. However, I think it's too long. So this change indicates just a property name.